lighthouse-cli/cli-flags.js

```diff
@@ -124,10 +124,10 @@ function getFlags(manualArgv, options = {}) {
           type: 'boolean',
           describe: 'Pause after page load to wait for permission to continue the run, evaluate `continueLighthouseRun` in the console to continue.',
         },
-        'fraggle-rock': {
+        'legacy-navigation': {
           type: 'boolean',
           default: false,
-          describe: '[EXPERIMENTAL] Use the new Fraggle Rock navigation runner to gather results.',
+          describe: '[DEPRECATED] Use the legacy navigation runner to gather results.',
```

Comment:
Say more about why you might want to use this?

Reply:
> I don't think we have any specific scenarios where FR runner would break

The new config format would break some clients. I'll add a note about that.

This might be a good time to update the config docs, so we know what our story will be.